GirafRest.IntegrationTest/Setup/WeekTemplateFixture.cs

```diff
@@ -1,41 +0,0 @@
-﻿using Newtonsoft.Json.Linq;
```

Comment:
Hvorfor har i slettet WeekTemplateFixture?

Reply:
WeekTemplate tests blev slettet for nogle commits siden, så den stod bare og samlede støv (samt også en mindre fixture jeg skal tilføje dipose methods til 😌)